src/main/kotlin/net/theevilreaper/dartpoet/DartFile.kt

```diff
@@ -97,7 +97,7 @@ class DartFile internal constructor(
             "The given path $path exists but it is not a directory"
         }
 
-        require(isDartConventionFileName(name)) {
+        require(!isDartConventionFileName(name)) {
```

Comment:
Why that be negated ?

Reply:
The pattern behind the method checks if the string matches with the dart file convetion. I need a negation for that because i want to send the message only if the string doesn't match with it (the implementation of require only throws an exception when the value is negated).